app/helpers/concerns/foreman_remote_execution/hosts_helper_extensions.rb

```diff
@@ -8,11 +8,27 @@ module HostsHelperExtensions
     end
 
     def multiple_actions_with_remote_execution
-      multiple_actions_without_remote_execution + [[_('Run Job'), new_job_invocation_path, false]]
+      multiple_actions_without_remote_execution + [ [_('Schedule Job'), new_job_invocation_path, false] ]
+    end
+
+    def schedule_job_multi_button(*args)
+      host_features = RemoteExecutionFeature.where(:host_action_button => true).map do |feature|
```

Comment:
what about using named scope `with_host_action_button` instead. And in adding a test into remote_execution_feature test to cover registration and retrieving of the action buttons?

BTW. what should be the ordering of the buttons?